modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java

```diff
@@ -145,6 +150,24 @@ public List<KeyValuePair> getScopeKeyValuePairs(String scope) {
 		return keyValuePairs;
 	}
 
+	public List<Map<String, String>> getStorageTypesMaps(Locale locale) {
```

Comment:
@murilostodolni since you are returning a `List` and not a `Map`, you can rename this method to `getStorageTypes`.